packages/flow-core/src/components/f-switch/f-switch.test.ts

```diff
@@ -0,0 +1,18 @@
+import { html, fixture, expect } from "@open-wc/testing";
+
+// import flow-core elements
+import "@cldcvr/flow-core";
+
+import { FSwitch } from "@cldcvr/flow-core";
+
+describe("f-switch", () => {
+  it("is defined", () => {
+    const el = document.createElement("f-switch");
+    expect(el).instanceOf(FSwitch);
+  });
+  it("should render with all default properties", async () => {
+    const el = await fixture(html` <f-switch></f-switch> `);
+    const descendant = el.shadowRoot!.querySelector(".f-switch")!;
+    expect(descendant.getAttribute("state")).to.equal("default");
+  });
```

Comment:
Can please add test case for testing `@input` event.

please Check [here](https://open-wc.org/docs/testing/helpers/#testing-events)